scripts/github-actions/create-pull-request.js

```diff
@@ -17,6 +17,30 @@ const createPullRequest = async ({ context, github, baseBranch, branchName, desc
       reviewers,
     })
   }
+
+  if (addToProjectBoard) {
+    const addToProjectBoardQuery = `
```

Comment:
I'd be curious to know why this [line](https://github.com/cypress-io/cypress/blob/0e239bf2108b6a80c0d4a5a4c017b77568101549/.github/workflows/triage_add_to_project.yml#L44) isn't handling these PRs. Does that workflow not get called when this PR is created?

Reply:
yeah I can't figure it out for the life of me, so I just was explicit about adding it.